js/skin.js

```diff
@@ -123,19 +122,30 @@ var Skin = React.createClass({
    * @public
    * @description the function is called when we stop the rotation
    */
-  handleVrPlayerMouseUp: function() {
+  handleVrPlayerMouseUp: function(e) {
     if (this.props.controller && this.props.controller.isVrStereo) {
       return;
     }
     if (this.props.controller && this.props.controller.videoVr) {
+      var isVrMouseMove = this.state.isVrMouseMove;
+      if (Utils.isIos()) {
+        isVrMouseMove = false;
```

Comment:
How come we are setting this to false on iOS?

Reply:
\+